### PR TITLE
[candi] in some cases /opt/deckhouse/bin not in the PATH fix #5043

### LINKED ISSUE #5043: [candi] in some cases /opt/deckhouse/bin not in the PATH
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

V1.46.10

### Expected Behavior

```
d.romanenko@master-0 ~ $ sudo su
root@master-0 /home/flant/d.romanenko # echo $PATH | grep -o "/opt/deckhouse/bin"
/opt/deckhouse/bin
```

### Actual Behavior

When we do `sudo su` from user `/opt/deckhouse/bin` path not in the PATH variable.
```
d.romanenko@master-0 ~ $ sudo su
root@master-0 /home/flant/d.romanenko # echo $PATH | grep -o "/opt/deckhouse/bin"
root@master-0 /home/flant/d.romanenko #
```

### Steps To Reproduce

1. Login to cluster node.
2. sudo su
3. echo $PATH | grep -o "/opt/deckhouse/bin"

### Additional Information

Proposed solution - add `/opt/deckhouse/bin` path to `/etc/environment`

### Logs

_No response_

### PR DESCRIPTION
**Pull request by Beastlex**
## Description

Fix issue with /opt/deckhouse/bin is not in $PATH for root user

## Why do we need it, and what problem does it solve?

We have problem when we do sudo su from user /opt/deckhouse/bin path not in the PATH variable.

## What is the expected result?

Correct usage binary tools on all flavours

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: Fixes when /opt/deckhouse/bin not in the PATH
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/static'...

**Comment by z9r5** [comment]
/e2e/abort static;Static;containerd;1.23 5799258570 5799258570-1-con-1-23 /sys/deckhouse-oss/install:pr5406 astra@192.168.199.12

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

File: candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl [hunk 1]
```diff
@@ -15,3 +15,18 @@
...
+#        fi
+# done
+# so /etc/profile.d/02-deckhouse-path.sh should be executable
+chmod a+x /etc/profile.d/02-deckhouse-path.sh
+
+# also need to sure that path to deckhouse bin in root bashrc
+if grep -q "^PATH=" /root/.bashrc 2>/dev/null; then
+  sed 's/^PATH=.*//' -i /root/.bashrc
+fi
+echo "export PATH=/opt/deckhouse/bin:$PATH" >> /root/.bashrc
```
Comment (RomanenkoDenys) [inline candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl:32]: This command will run every 4 hours.

Comment (RomanenkoDenys) [inline candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl:32]: This command will run every 4 hours.

**Review by RomanenkoDenys — COMMENTED** [review]
some nits

File: candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl [hunk 2]
```diff
@@ -15,3 +15,18 @@
...
 export PATH="/opt/deckhouse/bin:$PATH"
 EOF
+
+# On Alt Linux there is condition
+# for f in /etc/profile.d/*.sh; do
+#        if [ -f "$f" -a -r "$f" -a -x "$f" -a -s "$f" -a ! -L "$f" ]; then
+#                . "$f"
+#        fi
+# done
+# so /etc/profile.d/02-deckhouse-path.sh should be executable
```
Comment (RomanenkoDenys) [inline candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl:25]:
```suggestion
# On AltLinux /etc/profile.d/02-deckhouse-path.sh should be executable.
```

File: candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl (hunk 1 above)
Comment (name212) [reply candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl:26]: And file /root/.bashrc will increase size every 4 hours

**Review by name212 — CHANGES_REQUESTED** [review]
Need investigation

**Review by name212 — APPROVED** [review]

**Pull request merged**